Makefile

```diff
@@ -66,8 +66,8 @@ lint-all: lint-js lint-python lint-html lint-docs lint-docker lint-actions lint-
 
 lint: lint-js lint-python lint-html lint-links
 lint-js:
-	@yarn eslint --ext js --report-unused-disable-directives "**/*.{js,ts}"
-	@yarn prettier --check "**/*.{js,jsx,ts,tsx,mjs,cjs,mts,cts,md,sql,json,yml,html,css,scss}"
+	@yarn eslint --ext js --report-unused-disable-directives --cache --cache-strategy content "**/*.{js,ts}"
+	@yarn prettier "**/*.{js,jsx,ts,tsx,mjs,cjs,mts,cts,md,sql,json,yml,html,css,scss}" --check --cache --cache-strategy content
```

Comment:
This is borderline harmful in local dev, especially when working with Prettier/ESLint plugins, of which we have two: `packages/eslint-pluign-prairielearn` and `packages/prettier-plugin-sql`. Locally, I made changes to them that I knew would cause linting errors, and none were reported (because the caching mechanism doesn't know about those changes). This would be very frustrating for someone trying to iterate on changes to these plugins that either didn't know about or forgot about caching behavior.

I'd be OK if you wanted to conditionally enable caching in CI, where (at least for now) cache invalidation handles these cases. I'm not comfortable unconditionally enabling caching in local dev. I'd rather have no cache than broken cache invalidation.

Reply:
Resolved in e41a900a7fe941d58a7d970fc17e42d07a88db37 using a separate target.